Dockerfile

```diff
@@ -1,7 +1,11 @@
-FROM node:14
-WORKDIR /opt/spacebin-api
-COPY package.json yarn.lock ./
-COPY . .
-RUN yarn
-EXPOSE 7777
-CMD [ "yarn", "start" ]
+FROM node:latest
+
+RUN git clone https://github.com/spacebin-org/server.git spacebin
+
+WORKDIR spacebin
+
+RUN yarn add sqlite3
+
+EXPOSE 7777:7777
```

Comment:
Codacy found an issue: [Dockerfile:9:12:
  |
9 | EXPOSE 7777:7777
  |            ^
unexpected ':'
expecting a new line followed by the next instruction, at least one space, or digit
](https://app.codacy.com/gh/spacebin-org/server/pullRequest?prid=5760887)

Reply:
@jackmerrill